src/main/java/zd/zero/waifu/motivator/plugin/service/ApplicationService.kt

```diff
@@ -0,0 +1,11 @@
+package zd.zero.waifu.motivator.plugin.service
+
+import com.intellij.openapi.Disposable
+import com.intellij.openapi.application.ApplicationManager
+
+object ApplicationService : Disposable {
```

Comment:
I put this here because of a memory leak issue with the balloons.

See this PR for more details:
https://github.com/ChrisRM/material-theme-jetbrains/pull/1623